docker-app/qfieldcloud/subscription/models.py

```diff
@@ -437,6 +437,7 @@ class UpdateSubscriptionKwargs(TypedDict):
 
     plan = models.ForeignKey(
         Plan,
+        verbose_name=_("Plan"),
```

Comment:
Why did you add the verbose name? What it was showing without it?

Reply:
I added the verbose name for UI consistency, so that all filters have a capital letter: "By Period", "By Status", "By Plan", etc...
Otherwise it shows "By plan".

Reply:
less code, more fun :)
And TBH I don't see a special reason why we need the model to be capital in this context.